packages/amplify-graphql-model-transformer/src/resolvers/rds/query.ts

```diff
@@ -1,6 +1,11 @@
 import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
 import { compoundExpression, list, methodCall, obj, printBlock, qref, ref, set, str } from 'graphql-mapping-template';
-import { constructAuthFilterStatement, constructNonScalarFieldsStatement, constructFieldMappingInput } from './resolver';
+import {
+  constructAuthFilterStatement,
+  constructNonScalarFieldsStatement,
```

Comment:
We are still storing the array fields as part of the non-scalar fields metadata since adapters like MySQL do not need the information about which is array or object.